Formula/k/kubeone.rb

```diff
@@ -0,0 +1,39 @@
+class Kubeone < Formula
+  desc "Kubermatic KubeOne automates cluster operations on all your environments"
```

Comment:
```suggestion
  desc "Automate cluster operations on all your environments"
```

Reply:
Works for me, I just grabbed the one from the project desc. :)